src/utils/format.ts

```diff
@@ -15,27 +28,170 @@ function _getLocaleSymbols(): { thousands: string; decimals: string } {
 }
 
 const { thousands: THOUSANDS_SYMBOL, decimals: DECIMALS_SYMBOL } = _getLocaleSymbols()
-const DEFAULT_THOUSANDS_SYMBOL = ','
-const DEFAULT_DECIMALS_SYMBOL = '.'
+
+const _formatDecimalsFromWeiForDisplay = (decimalsToConvert: BN, decimalSymbol: string = DEFAULT_DECIMALS_SYMBOL) => '0' + decimalSymbol + fromWei(decimalsToConvert).slice(2)
```

Comment:
I think this one-line expressions are meant for simple definitions 
This one is a bit hard to read (difficult name, then types, and theres's sth odd in that fromWei/slice. 
Can you make it a normal function?

Reply:
done